style.css

```diff
@@ -0,0 +1,189 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+html, body {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: 'Poppins';
+}
+
+h1 {
+    font-size: 5em;
+    font-weight: 900;
+}
+
+h3 {
+    background-color: rgba(255,204,53,.4);
+    padding: 0 30px;
+    transform: skew(9deg,0deg);
+    font-style: italic;
+    font-weight: bold;
+    font-size: 2.5em;
+}
+
+h4 {
+    color: #ED1C24;
+}
+
+header > div {
+    padding: 0 25px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid lightgray;
+}
+  
+nav {
+    width: 600px;
+}
+  
+nav ul {
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+  
+nav a {
+    text-decoration: none;
+    color: black;
+}
+  
+.blackHeart {
+    width: 20px;
+    margin-right: 1rem;
+}
+
+.logo {
+    width: 80px;
+    margin-right: 1rem;
+}
+
+.search-bar {
+    width: 30px;
+    height: auto;
+    background-color: rgba(0,0,0,.05);
+    padding: 10px 15px;
+}
+
+.search-button input {
+    width: 950px;
+    padding: 3px;
+    border: none;
+    background-color: rgba(0,0,0,.05);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 1em;
+}
+
+.search-box {
+    border: none;
+    padding: 0 20px;
+    background-color: #fb3e44;
+    color: white;
+    cursor: pointer;
+    font-size: 15px;
+}
+
+
+form {
+    display: flex;
+    padding: 10px 0;
+
+}
+
+.link-btn {
+    background-color: white;
+    color: black;
+    text-align: center;
+    text-decoration: none;
+    padding: 7px 30px;
+}
+
+#join-button {
+    border: 1px solid black;
+}
+
+#cream-background {
+    background-color: rgba(0,0,0,.05);
+    height: 550px;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    padding: 0 0 80px 0;
+}

```

Comment:
The top and bottom of this section seems a bit too big. Since you have no margins or padding there, you can tell it's the H1's default margin. You can decrease it and then adjust the padding-bottom accordingly.